### PR TITLE
Removed H4s

### PR DESCRIPTION
**Pull request by Maggers**
Updated ms.date to 8/9/2021.

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md [hunk 1]
```diff
@@ -54,29 +54,25 @@ Learn more about using protocol handlers with your PWA at [Experimental features
...
 When a user selects the **Settings** (**...**) menu in the app's title bar, the _app info_ menu is shown. We've updated this menu and streamlined the experience for the user in the following ways:
 * Moved the app publisher information to the top level and made it the first thing a user sees.
-* Moved the privacy information and controls into a dedicated 2nd-level **Privacy** menu.
-* Moved content-related tools into a dedicated 2nd-level **More tools** menu.
-
-These changes make the **Settings** menu provide a user experience that's more like a desktop app than a browser UI.
-
-#### Updated App Info Menu
 
 :::image type="complex" source="media/app-info.png" alt-text="The new, streamlined app-info menu" lightbox="media/app-info.png":::
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md:58]: Add leading spaces to indent the images, to be presented as a child of each list item.   
- For list items, in my experiment, I put * in column 1, then text of the list item started ~column 4.
- For each image, I selected all lines of an image and pressed Tab key to add maybe 4 leading spaces, to match the indent of the text of the list item.  
Then I viewed Preview in VS Code (Sh+Ctrl+V) and it looked good there.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Indent images

File: microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md (hunk 1 above)
Comment (Maggers) [reply microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md:58]: Ok updated.

File: microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md [hunk 2]
```diff
@@ -53,37 +53,33 @@ Learn more about using protocol handlers with your PWA at [Experimental features
 ## Streamlined app-info menus
 
 When a user selects the **Settings** (**...**) menu in the app's title bar, the _app info_ menu is shown. We've updated this menu and streamlined the experience for the user in the following ways:
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md:55]:
```suggestion
When a user selects the **Settings** (**...**) menu in the app's title bar, the **App info** menu is displayed.  We've updated the **App info** menu and streamlined the user experience in the following ways, to provide a user experience that's more like a desktop app than a browser UI:
```

File: microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md [hunk 3]
```diff
@@ -53,37 +53,33 @@ Learn more about using protocol handlers with your PWA at [Experimental features
...
 
-#### Dedicated More Tools menu
+   :::image type="complex" source="media/more-tools.png" alt-text="Content-related tools are now found in the More Tools menu" lightbox="media/more-tools.png":::
+      Content-related tools are now found in the **More tools** menu
+   :::image-end:::
 
-:::image type="complex" source="media/more-tools.png" alt-text="Content-related tools are now found in the More Tools menu" lightbox="media/more-tools.png":::
-   Content-related tools are now found in the **More tools** menu
-:::image-end:::
+These changes make the **Settings** menu provide a user experience that's more like a desktop app than a browser UI.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md:74]:
```suggestion
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**